routing/matcher.go

```diff
@@ -96,12 +81,14 @@ func (err *definitionError) Error() string {
 	return fmt.Sprintf("%s [%d]: %v", err.Id, err.Index, err.Original)
 }
 
+// rx identifying the 'free form' wildcards at the end of the paths
 var freeWildcardRx *regexp.Regexp
 
 func init() {
 	freeWildcardRx = regexp.MustCompile("/[*][^/]+$")
```

Comment:
Is there a difference from putting it directly in the var line above, since this is using `MustCompile()`...